### PR TITLE
Installing and using wrld-react

### PR DESCRIPTION
**Pull request by calumleask**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,113 +41,59 @@ In a new directory where your project will be contained, run:
 npm init react-app <project-name-here>
 ```
 
-#### Install wrld.js
+#### Install wrld-react
 
 ```sh
```
Comment (ghost) [inline README.md:46]: Need to make sure the user cd's into <project-name-here> before they install, I think?

File: example/package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
     "react-dom": "^17.0.1",
     "react-scripts": "4.0.0",
     "web-vitals": "^0.2.4",
+    "wrld-react": "^0.1.2261",
     "wrld.js": "^0.1.2255"
```
Comment (ghost) [inline example/package.json]: Can we do without a dependency on wrld.js, as it's already a dependency of wrld-react (and it saves having two version numbers specified explicitly)

**Review by ghost — CHANGES_REQUESTED** [review]
Couple of minor changes to suggest, but all works fine when I tested it out 👏👏👏

File: example/package.json (hunk 2 above)
Comment (calumleask) [reply example/package.json]: Forgot to remove it. Done now

File: README.md (hunk 1 above)
Comment (calumleask) [reply README.md:46]: They don't need to cd into anything here. These are instruction how to set up your own app and install wrld-react

**Review by ghost — APPROVED** [review]

**Pull request merged**